spec.md

```diff
@@ -2606,6 +2742,24 @@ enumerate(["zero", "one", "two"])               # [(0, "zero"), (1, "one"), (2,
 enumerate(["one", "two"], 1)                    # [(1, "one"), (2, "two")]
 ```
 
+### float
+
+`float(x)` interprets its argument as a floating-point number.
+
+If x is a `float`, the result is x.
+
+If x is an `int`, the result is the floating-point value nearest x.
+The call fails if x is too large to represent as a `float`.
+
+If x is a `bool`, the result is `1.0` for `True` and `0.0` for `False`.
+
+If x is a string, the string is interpreted as a floating-point literal.
+The function also recognizes the names `Inf` (or `Infinity`) and `NaN` of
+the IEEE 754 non-finite values, optionally preceded by a `+` or `-` sign.
```

Comment:
Nit: "The function also recognizes the names `Inf` (or `Infinity`) and `NaN`, optionally proceeded by a `+` or `-` sign. These construct the IEEE 754 non-finite values." (Avoids implication that there are only two non-finite values or that -inf and +inf share a name.)

Reply:
Better. Done.